gravitee-apim-console-webui/src/management/api/documentation-v4/components/documentation-home-page-header/api-documentation-v4-home-page-header.component.html

```diff
@@ -17,9 +17,17 @@
 -->
 <h3 class="header__title">Homepage</h3>
 <div class="header__action" *gioPermission="{ anyOf: ['api-documentation-c'] }">
-  <button [disabled]="true" class="header__action__button" mat-stroked-button>Choose Existing Page</button>
+  <button
+    [disabled]="hasHomepage || noCustomPages || isReadOnly"
+    class="header__action__button"
+    mat-stroked-button
+    (click)="selectPage.emit()"
+    *gioPermission="{ anyOf: ['api-documentation-u'] }"
+  >
+    Choose Existing Page
+  </button>
   <api-documentation-v4-add-page-button
-    [disabled]="hasPages"
+    [disabled]="hasHomepage || isReadOnly"
     (addPage)="addPage.emit($event)"
     text="Create New Page"
   ></api-documentation-v4-add-page-button>
```

Comment:
If a homepage is set, the buttons are no longer shown.

Reply:
Done